python/pyspark/sql/connect/client.py

```diff
@@ -42,6 +43,125 @@
 logging.basicConfig(level=logging.INFO)
 
 
+class ChannelBuilder:
+    """
+    This is a helper class that is used to create a GRPC channel based on the given
+    connection string per the documentation of Spark Connect.
+    """
+
+    PARAM_USE_SSL = "use_ssl"
+    PARAM_TOKEN = "token"
+    PARAM_USER_ID = "user_id"
```

Comment:
Is this case sensitive?

Reply:
the comparison is done case-insensitive later.

Reply:
Is this a well known thing for connection string? If not can we BTW update the doc to mention this?

Reply:
I will update the doc.

Reply:
@amaliujia https://github.com/apache/spark/pull/38501